RecoParticleFlow/PFClusterProducer/python/particleFlowClusterOOTECAL_cff.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
-from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cff import *
 
 particleFlowClusterOOTECAL = particleFlowClusterECAL.clone()
+particleFlowClusterOOTECAL.energyCorrector.applyMVACorrections = True
+particleFlowClusterOOTECAL.energyCorrector.maxPtForMVAEvaluation = 90.
```

Comment:
Sorry, had to step out of the office... I will commit in a bit. That's the last piece left.